keyboards/custommk/ergostrafer/rules.mk

```diff
@@ -0,0 +1,4 @@
+EEPROM_DRIVER = spi
+
+AUDIO_ENABLE = yes           # Audio output
```

Comment:
```suggestion
```

Reply:
I had missed that audio enabling could now be in info.json (wasn't in documentation yet), fixed.